packages/core/stories/about/Introduction.stories.mdx

```diff
@@ -5,60 +5,104 @@ import {
 
 import '../styles/index.scss';
 
-
 import generateTitle from '../util/generateTitle';
 
 <Meta title={generateTitle('Introduction')} />
 
 # Welcome to Mayflower
 
-Mayflower is a living styleguide for the Massachusetts state government — providing guidance and tools for Massachusetts state agencies to create a cohesive government identity and trustworthy look and feel across Commonwealth digital ecosystem.
-
-The goal for Mayflower styleguide is to to help Massachusetts state organizations to easily and quickly apply the Commonwealth branding, follow to best practices, and create a consistent and delightly digital experience regardless of the technology stacks. Refer to the following three areas to "Mayflowerize" your product:
-
-- **Design principles** are the guiding theories and rationales for the Mayflower brand and other implementation ground rules, where all the system-wide design decisions and best practices are rooted in.
-
-- **Design tokens** are assets and variables referenced and propagated throughout the design system.
+Mayflower is the design system for the government of the state of Massachusetts. It's a tool state agencies can use to develop and maintain digital experiences that provide a consistent and trustworthy look and feel across our digital ecosystem.
 
-- **Branding components** are the templating components, such as Headers and Footers, and other base elements, such as Heading, Links, Buttons, that are frequently used in all web products.
+Read more below for in-depth information about the stystem or go to [Quick Start](quickstart) to dive right in and start using Mayflower.
 
-<div className="row ma__row--three-up">
+<!-- This is using the full markup because links will not work with a generated component (as far as I can tell) -->
+<div className="sandbox row ma__row--three-up">
   <div className="col-md">
-    <SectionLinks
-      title={{
-        text: 'Design Principles'
-      }}
-      description="Follow these high-level guides when making decisions on design and development of any digital or print products, and when adopting or contributing to Mayflower."
-      index={0}
-    >
-      <DecorativeLink text="Brand Pillars" href="" />
-      <DecorativeLink text="Design Guidelines" href="" />
-    </SectionLinks>
+    <section className="ma__section-links">
```

Comment:
why not use the mayflower react component for SectionLinks?

Reply:
When I did that I could not get links to other pages in docs to work. Is there a special syntax for that?

Reply:
Ah. I found it. The documentation is in the addon

Reply:
Ok, pushed a new commit with LinkTo implemented in the MDX file.

Reply:
I think you could just use LinkTo inside of <SectionLinks>. Trying that now, if that works I will push it up